modules/mlflow/mlflow-fargate/stack.py

```diff
@@ -0,0 +1,206 @@
+        )
+        self.cluster = cluster
+
+        task_definition = ecs.FargateTaskDefinition(
+            self,
+            "MlflowTask",
+            task_role=role,
+            cpu=task_cpu_units,
+            memory_limit_mib=task_memory_limit_mb,
+        )
+
+        container = task_definition.add_container(
+            "ContainerDef",
+            image=ecs.ContainerImage.from_ecr_repository(
+                repository=ecr.Repository.from_repository_name(
+                    self,
+                    "ECRRepo",
+                    repository_name=ecr_repo_name,
+                ),
+            ),
+            environment={
+                "BUCKET": f"s3://{artifacts_bucket_name}",
+                "DUMMY": "DUMMY",
+            },
+            logging=ecs.LogDriver.aws_logs(stream_prefix="mlflow"),
+        )
+        port_mapping = ecs.PortMapping(container_port=5000, host_port=5000, protocol=ecs.Protocol.TCP)
+        container.add_port_mappings(port_mapping)
+
+        # Add EFS
+        fs = efs.FileSystem(
+            self,
+            "EfsFileSystem",
+            vpc=vpc,
+            encrypted=True,
+            throughput_mode=efs.ThroughputMode.ELASTIC,
+            performance_mode=efs.PerformanceMode.GENERAL_PURPOSE,
+            file_system_policy=iam.PolicyDocument(
+                statements=[
+                    iam.PolicyStatement(
+                        actions=[
+                            "elasticfilesystem:ClientMount",
+                            "elasticfilesystem:ClientWrite",
+                            "elasticfilesystem:ClientRootAccess",
+                        ],
+                        principals=[iam.AnyPrincipal()],
+                        resources=["*"],
+                        conditions={"Bool": {"elasticfilesystem:AccessedViaMountTarget": "true"}},
+                    ),
+                ]
+            ),
+        )
+        self.fs = fs
+
+        # Attach and mount volume
+        task_definition.add_volume(
+            name="efs-volume",
+            efs_volume_configuration=ecs.EfsVolumeConfiguration(
+                file_system_id=fs.file_system_id,
+                transit_encryption="ENABLED",
+            ),
+        )
+        container.add_mount_points(
+            ecs.MountPoint(
+                container_path="./mlruns",
+                source_volume="efs-volume",
+                read_only=False,
+            )
+        )
+
+        # Create ECS Service
+        service = ecs_patterns.NetworkLoadBalancedFargateService(
+            self,
+            "MlflowLBService",
+            service_name=service_name,
+            cluster=cluster,
+            task_definition=task_definition,
+            task_subnets=ec2.SubnetSelection(subnets=subnets),
+            circuit_breaker=ecs.DeploymentCircuitBreaker(rollback=True),
+        )
+        self.service = service
+
+        # Enable access logs
+        lb_access_logs_bucket = s3.Bucket(
```

Comment:
Can you consume this [bucket](https://github.com/awslabs/idf-modules/tree/main/modules/storage/buckets) and store access logs instead of creating one here?

Reply:
Thanks! Added parameters to provide access logs bucket/prefix